tests/utils/axis-utils-tests.js

```diff
@@ -32,7 +32,7 @@ test('axis-utils #getTicksTotalFromSize', t => {
 
 test('axis-utils #getTickValues', t => {
   const scale = scaleLinear().domain([0, 1]).range(['red', 'blue']);
-  t.deepEqual(getTickValues(scale, 10, false),
+  t.deepEqual(getTickValues(scale, 10, false).map(d => Math.round(d * 1000) / 1000),
```

Comment:
:( I was hoping that balthazar's change was related to https://twitter.com/mbostock/status/853254045477126144 

(which in turn i was hoping had resolved https://github.com/uber/react-vis/issues/230), but alas. I think tick values should probable just accepts another argument called precision or something